client/src/App.jsx

```diff
@@ -53,11 +53,17 @@ export default function App() {
 
         // BUG: SOMETIMES THERE ARE DUPLICATES!!!!!!!!!
         // Filtered on nationality because then the names get too hard :p
-        const apiParams = "?format=JSON&nat=CA,US&results=" + numOfRandomUsers
+        
+        
+        const apiParams = "?format=JSON&nat=CA,US&results=" + (numOfRandomUsers * 2)
         tryFetchData(RANDOM_USER_GENERATOR_API_URL + apiParams)
             .then((data) => {
-                setRandomUsers(data.results);
+                console.log(data.results)
+                const randomUsers = checkImageDuplicates(data.results)
+                setRandomUsers(randomUsers);
             })
+        
```

Comment:
how come you like to add new lines

Reply:
It spaces things out and it looks more readable to me